CHANGE-NOTES.html

```diff
@@ -6,4 +6,5 @@
   <li>Added help action</li>
   <li>Improved displaying of status</li>
   <li>Improved push action to automatically suggest force-push when needed</li>
+  <li>Improved stability of push dialog (v0.5.1)</li>
```

Comment:
I think we can approach change-notes in two ways, depending on the case:
1. either scrap the content completely and replace with new changelog - makes sense for minor&major releases
1. or just append new pieces of changelog - makes sense for patch releases like this one

Reply:
The automated checks in CI only require that the CHANGE-NOTES.html are _modified_ compared to master.